e2e/features/exit-appeal-from-the-online-service.feature

```diff
@@ -0,0 +1,86 @@
+Feature: End appeal
```

Comment:
`Exit online appeal` ?

Reply:
To avoid confusion renamed it to 'remove'